### PR TITLE
Runtime 2800: Governance V1 removal and addition of Fast General Admin Track

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR addresses the governance-related changes in runtime 2800:

- It removes the Governance V1 collectives: Council and Tech Committee
- It adds the Fast General Admin Track configs - this is for Moonbase Alpha only

Note: there is a link in this PR to a new page that was created in a separate PR that is under review (https://github.com/moonbeam-foundation/moonbeam-docs/pull/877)

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/build/canonical-contracts.md [hunk 1]
```diff
@@ -59,88 +59,88 @@ There are a set of precompiled contracts included on Moonbeam, Moonriver, and Mo
...
+    |:-------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------:|
+    |      [Parachain Staking](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/parachain-staking/StakingInterface.sol){target=\_blank}      |              {{networks.moonbeam.precompiles.staking}}              |
+    |     [Crowdloan Rewards](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/crowdloan-rewards/CrowdloanInterface.sol){target=\_blank}     |             {{networks.moonbeam.precompiles.crowdloan}}             |
+    |             [ERC-20 Interface](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/balances-erc20/ERC20.sol){target=\_blank}              |               {{networks.moonbeam.precompiles.erc20}}               |
+    |    [Democracy [Removed]](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/pallet-democracy/DemocracyInterface.sol){target=\_blank}     |             {{networks.moonbeam.precompiles.democracy}}             |
```
Comment (albertov19) [inline builders/build/canonical-contracts.md:67]: It was actually just removed in Moonbase now, we will do a release per network?

File: builders/build/canonical-contracts.md [hunk 2]
```diff
@@ -59,88 +59,88 @@ There are a set of precompiled contracts included on Moonbeam, Moonriver, and Mo
...
+    |:-------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------:|
+    |      [Parachain Staking](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/parachain-staking/StakingInterface.sol){target=\_blank}      |              {{networks.moonbase.precompiles.staking}}              |
+    |     [Crowdloan Rewards](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/crowdloan-rewards/CrowdloanInterface.sol){target=\_blank}     |             {{networks.moonbase.precompiles.crowdloan}}             |
+    |             [ERC-20 Interface](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/balances-erc20/ERC20.sol){target=\_blank}              |               {{networks.moonbase.precompiles.erc20}}               |
+    |         [Democracy](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/pallet-democracy/DemocracyInterface.sol){target=\_blank}          |             {{networks.moonbase.precompiles.democracy}}             |
```
Comment (albertov19) [inline builders/build/canonical-contracts.md:125]:
```suggestion
    |         [Democracy [Removed]](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/pallet-democracy/DemocracyInterface.sol){target=\_blank}         |             {{networks.moonbase.precompiles.democracy}}             |
```

File: builders/build/canonical-contracts.md (hunk 1 above)
Comment (albertov19) [inline builders/build/canonical-contracts.md:67]:
```suggestion
    |    [Democracy](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/pallet-democracy/DemocracyInterface.sol){target=\_blank}     |             {{networks.moonbeam.precompiles.democracy}}             |
```

File: builders/pallets-precompiles/precompiles/overview.md [hunk 3]
```diff
@@ -34,88 +34,88 @@ The precompiled contracts are categorized by address and based on the origin net
...
+    |:-------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------:|
+    |      [Parachain Staking](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/parachain-staking/StakingInterface.sol){target=\_blank}      |              {{networks.moonbeam.precompiles.staking}}              |
+    |     [Crowdloan Rewards](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/crowdloan-rewards/CrowdloanInterface.sol){target=\_blank}     |             {{networks.moonbeam.precompiles.crowdloan}}             |
+    |             [ERC-20 Interface](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/balances-erc20/ERC20.sol){target=\_blank}              |               {{networks.moonbeam.precompiles.erc20}}               |
+    |    [Democracy [Removed]](https://github.com/moonbeam-foundation/moonbeam/blob/master/precompiles/pallet-democracy/DemocracyInterface.sol){target=\_blank}     |             {{networks.moonbeam.precompiles.democracy}}             |
```
Comment (albertov19) [inline builders/pallets-precompiles/precompiles/overview.md:42]: Same as before

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks good! Left some feedback to clarify Democracy V1 removal

File: builders/build/canonical-contracts.md (hunk 1 above)
Comment (eshaben) [reply builders/build/canonical-contracts.md]: Actually [as of runtime 2801 it hasn't been removed on Moonbase](https://github.com/moonbeam-foundation/moonbeam/blob/runtime-2801/runtime/moonbase/src/precompiles.rs) yet. I added a spec_version tag so for the next release, i'll be reminded to check it and remove it

**Review by albertov19 — APPROVED** [review]
LGTM!

**Pull request merged**